openstf/feature_engineering/general.py

```diff
@@ -76,3 +80,37 @@ def remove_extra_feature_columns(
         )
 
     return input_data.drop(extra_features, axis=1)
+
+
+def enforce_feature_order(df):
```

Comment:
More in general. Why do we need this function, could we not just make sure that our input_data is always ordered properly instead of fixing it afterwards? Which part of the code determines the original order and why is that different from what we need later on?